source/solvers/gd_navier_stokes.cc

```diff
@@ -60,6 +60,13 @@ GDNavierStokesSolver<dim>::setup_assemblers()
 {
   this->assemblers.clear();
 
+  // Buoyant force
+  if (this->simulation_parameters.multiphysics.buoyancy_force)
```

Comment:
so you want to use the `buoyancy_force` even if VOF is not used?

Reply:
Buoyancy force should exists even when VOF is not used. Buoyancy and VOF are not related.